requirements.txt

```diff
@@ -1,4 +1,6 @@
 numpy
 matplotlib
 basemap
-psutil
\ No newline at end of file
+psutil
+brahe
```

Comment:
Why are we reusing brahe?

Reply:
possibly no longer needed, need to check